src/themes/teams/components/Chat/chatVariables.ts

```diff
@@ -0,0 +1,7 @@
+export interface IChatVariables {
+  backgroundColor: string
```

Comment:
this is better to be handled by the code of examples for now - more than that, we could adjust padding aspects there as well

Reply:
Actually, I added the backgroundColor and the padding to the variables with some defaults that would match the Team's theme, as we are developing this theme, so why not have this look by default. Please share your thoughts on this. Also, I added avatar slot in the ChatMessage, for configuring the statusBorderColor to match the background. Actually I have one thing that I am not 100% sure that should be implement this way. The Chat component has it's own backgroundColor and this color should match the Avatar's statusBorderColor. The problem is that, the Avatar is inside the ChatMessage component, and I am not confident that the Chat component should define the avatar's slot in the ChatMessage component. For now, those colors are independent variables in the Chat, and the ChatMessage's avatar slot component. What do you think about this @kuzhelov? This is how the default look of the examples looks now:
![image](https://user-images.githubusercontent.com/4512430/45753347-c418d980-bc18-11e8-966c-a640da6b0a55.png)

Reply:
agreed to not introduce these two as variables for now, just inline them into styles directly

Reply:
We are going to stick with the backgroundColor as variable in order to reuse the siteVariables. Paddings are deleted from variables